ako-gateway-api/status/status.go

```diff
@@ -80,7 +84,7 @@ func BulkUpdate(key string, objectType string, options []status.StatusOptions) e
 	return nil
 }
 
-func Record(key string, obj runtime.Object, status *Status) {
+func Record(key string, obj runtime.Object, status *Status) (runtime.Object, error) {
```

Comment:
@pkoshtavmware: I think we are not using return value `runtime.Object` anywhere. Can we take it out (in nested calls also)?

Reply:
Done